### DISCUSSION TITLE
How to increase the character limit?

### DISCUSSION THREAD

**Discussion by Elfstern** [speech-translator-ext/speech-translator-readme#19] [Q&A]
Hi all

Can I increase the character limit with the Google Translate API key to, for example, 200,000 characters per day?

Does the Speech Translator from kappaflow also support the API key from deepl?

Many thanks for your answers.

**Comment by kappaflow** [comment]
Hi!
> Can I increase the character limit with the Google Translate API key to, for example, 200,000 characters per day?

Do you experience an issue with the character limit on translation? Based on my experience, even using the extension all day long with _Google Translate (Public)_ (with a dedicated public IP address), I didn't get any limitations...

Another way to increase the limit is to use _Google Translate (Script)_. Check the limits [here]( https://developers.google.com/apps-script/guides/services/quotas).

If you want to use your custom API keys with different translation services (including deepl) you should request a custom extension build, which is available for [subscribers](https://boosty.to/speech-translator-ext) with `Super VIP Supporter` plan. For more details, please contact me by email that you can find [here](https://github.com/speech-translator-ext) or on [Boosty](https://boosty.to/speech-translator-ext).
